lib/sshutils/scp/scp.go

```diff
@@ -174,20 +175,18 @@ func CreateUploadCommand(cfg Config) (Command, error) {
 func (c *Config) CheckAndSetDefaults() error {
 	logger := c.Log
 	if logger == nil {
-		logger = log.StandardLogger()
-	}
-	c.Log = logger.WithFields(log.Fields{
-		teleport.ComponentKey: "SCP",
-		teleport.ComponentFields: log.Fields{
```

Comment:
Should we keep this in a group? TBH I've never paid attention to this in our use of logrus.